deploy/manifests/nginx-gateway.yaml

```diff
@@ -119,6 +119,7 @@ spec:
             fieldRef:
               fieldPath: status.podIP
         args:
+        - control-plane
```

Comment:
Can this be specified as the `command`. Like this: https://github.com/projectcontour/contour/blob/b28dae15469c4283a786dca0c5b859e5e301c71f/examples/deployment/03-envoy-deployment.yaml#LL64C11-L65C20

Reply:
this is a path to binary.

```
        command:
        - static-mode
        args:
        - --gateway-ctlr-name=k8s-gateway.nginx.org/nginx-gateway-controller
        - --gatewayclass=nginx
 ```
```
  Warning  Failed     11s (x3 over 40s)  kubelet            Error: failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "static-mode": executable file not found in $PATH: unknown
```

Reply:
ok, bummer. Thanks for checking